### DISCUSSION TITLE
desktop application is not running in wsl win10

### DISCUSSION THREAD

**Discussion by harmanpreet7916** [organicmaps/organicmaps#4472] [Q&A]

**Comment by harmanpreet7916** [comment]
sudo strip --remove-section=.note.ABI-tag /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 
this commad helped me

**Reply by rtsisyk** [reply]
Could you please update https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md?

**Reply by biodranik** [reply]
Why is this fix needed at all? What is the core issue?
